include/ignition/gazebo/components/Model.hh

```diff
@@ -29,14 +32,64 @@ namespace gazebo
 {
 // Inline bracket to help doxygen filtering.
 inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace serializers
+{
+  class SdfModelSerializer
+  {
+    /// \brief Serialization for `sdf::Model`.
+    /// \param[in] _out Output stream.
+    /// \param[in] _time Model to stream
+    /// \return The stream.
+    public: static std::ostream &Serialize(std::ostream &_out,
+                const sdf::Model &_model)
+    {
+      sdf::ElementPtr modelElem = _model.Element();
+      if (!modelElem)
+      {
+        ignerr << "Unable to serialize sdf::Model" << std::endl;
+        return _out;
+      }
+
+      _out << "<?xml version=\"1.0\" ?>"
+           << "<sdf version='" << modelElem->OriginalVersion() << "'>"
```

Comment:
The `OriginalVersion()` is the version of the file before it is (potentially) upconverted by the parser to the latest version. However, `ToString()` outputs the upconverted version, so I would say we should use `SDF_PROTOCOL_VERSION ` here instead.

Reply:
Thanks, replaced in de79f9a